signature-aggregator/api/api.go

```diff
@@ -0,0 +1,155 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package api
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"strings"
+
+	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/ava-labs/awm-relayer/signature-aggregator/aggregator"
+	"github.com/ava-labs/awm-relayer/types"
+	"github.com/ava-labs/awm-relayer/utils"
+	"go.uber.org/zap"
+)
+
+const (
+	RawMessageAPIPath = "/aggregate-signatures/by-raw-message"
```

Comment:
I'm a fan of single word paths, as long as they're descriptive enough within the context of the API. Here, I think `aggregate/message` or `signatures/message` would be appropriate. It's a matter of opinion of course, so lmk what you think.

Reply:
I agree that I'd like to update this. This is really a vestige of the old way where initial implementation supported both full message bytes as well as a `message id` so referencing `by-raw-message` was to distinguish from `by-id`. But we've since removed that. 

Since IIRC ACP-118 requires either the justification or message field to be non-zero I'd like to remove message entirely from the path. Any ideas about the most logical path here in that case `signatures/aggregate` ?

Reply:
good with `signatures/aggregate`

Reply:
I wrote the path that's in place here right now, but I'm liking these discussion directions more.

Some more food for thought:
* Should the path be a noun or a verb? I would think a noun. 
* Should it be `signature` (singular) or `signatures` (plural)? Seems like plural makes sense if we're using "aggregate" as a verb ("aggregate the signatures for me") but singular makes sense if we're using "aggregate" as a noun ("get me the aggregate signature").

My current best suggestion is just `/aggregate`. That leaves it open ended so that we don't have to answer any of those questions 😅 but if we want to add another word or two to the path then we should consider noun/verb/singular/plural.

Reply:
Friendly reminder to update the path references in the README as well :)

Reply:
If there's no intention to add different subpaths, I agree with @feuGeneA that a single path level is sufficient. I like `aggregate`, though if that's not descriptive enough, the original `aggregate-signatures` (without `by-raw-message`) is good too.

Reply:
+1 for me for `aggregate-signatures`